isso/js/app/templates/postbox.js

```diff
@@ -10,7 +10,7 @@ var html = function (globals) {
 "<div class='isso-postbox'>"
  + "<div class='isso-form-wrapper'>"
    + "<div class='isso-textarea-wrapper'>"
-    + "<div class='isso-textarea isso-placeholder' contenteditable='true'>" + i18n('postbox-text') + "</div>"
+    + "<textarea class='isso-textarea' rows='10' maxlength='65535' placeholder='" + i18n('postbox-text') + "'></textarea>"
```

Comment:
a) I can understand why you set **rows to 10** here - since we don't have auto-resizing (yet), you want the textarea to be big enough for people to enter medium-length text. But that makes the area quite long; especially on mobile already half a page length is taken up only by the postbox and it's not immediately clear that there will be actual comments when scrolling further down. Let's stick with an initial **5 rows** for now.

b) Why not add a `minlength` as well? let the browser do the validation instead of us having to do that. Should also have the benefit of localized error/hint messages instead of us having to carry translations.